config.ini

```
@@ -51,6 +51,11 @@ motd=Welcome to OpenFusion!
 # and pre-Academy builds must *not* contain it.
 #enabledpatches=1013
 
+# Use Original FusionFall's racing score and reward calculation?
+# Set false to use Retro's calculation, make sure you have the correct
+# patch(es) loaded.
+ogracingscores=true
```

you can comment this out since true is the default value
```suggestion
#ogracingscores=true
```